tools/shared.py

```diff
@@ -2908,6 +2908,9 @@ def run_binaryen_command(tool, infile, outfile=None, args=[], debug=False, stdou
       cmd += ['--input-source-map=' + infile + '.map']
       cmd += ['--output-source-map=' + outfile + '.map']
       cmd += ['--output-source-map-url=' + Settings.SOURCE_MAP_BASE + os.path.basename(Settings.WASM_BINARY_FILE) + '.map']
+      # remove any dwarf debug info sections, as the source map has the info,
+      # so they are not needed (and may be very large)
+      cmd += ['--strip-dwarf']
```

Comment:
Any reason not to just use `--strip-debug` here?

Reply:
It would do too much. In a source maps build, we do want the SourceMapUrl section, which strip-debug would remove.